routes/definitions.js

```diff
@@ -17,12 +17,20 @@ async function getDefinition(request, response) {
   const coordinates = utils.toEntityCoordinatesFromRequest(request)
   const pr = request.params.pr
   // if running on localhost, allow a force arg for testing without webhooks to invalidate the caches
-  const force = request.hostname.includes('localhost') ? request.query.force || false : false
+  const force = (request.hostname && request.hostname.includes('localhost')) ? request.query.force || false : false
   const expand = request.query.expand === '-files' ? '-files' : null // only support '-files' for now
   const result = await definitionService.get(coordinates, pr, force, expand)
+  response.set('cache-tag', definitionService.tagFromCoordinates(result))
```

Comment:
in order to build the hash correctly this should be 

```
response.set('cache-tag', definitionService.tagFromCoordinates(result.coordinates))
```

Reply:
I understand, you’re right; otherwise any tool additions will make it a different hash. Thanks for catching this!